Dockerfile.dev

```diff
@@ -0,0 +1,45 @@
+FROM debian:bullseye
+
+# Install OS Packages
+RUN apt update
+RUN apt install curl ca-certificates gnupg build-essential git -y
+
+# Install Postgres
+# Update postgresql APT repository [apt.postgresql.org](https://wiki.postgresql.org/wiki/Apt)
+RUN ["bash", "-c", "curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg >/dev/null"]
+RUN ["bash", "-c", "echo 'deb http://apt.postgresql.org/pub/repos/apt/ bullseye-pgdg main' > /etc/apt/sources.list.d/postgresql.list"]
+RUN apt update
+RUN apt upgrade -y
+RUN apt install postgresql-client-14 postgresql-14 postgresql-14-cron pgbouncer postgresql-server-dev-14 libcurl4-openssl-dev git make g++ locales-all -y
+
+RUN git clone --depth 1 https://github.com/pramsey/pgsql-http
+RUN cd pgsql-http && make && make install
+
+# Install NodeJS
+RUN ["bash", "-c", "curl -fsSL https://deb.nodesource.com/setup_16.x | bash -"]
+RUN apt install nodejs -y
+RUN npm install -g yarn
+
+# Install App
+WORKDIR /engine
+
+COPY package.json package.json
+COPY yarn.lock yarn.lock
+RUN yarn install
+
+COPY . .
+RUN yarn build
+
+COPY ./src/scripts/postgresql.conf /etc/postgresql/14/main/postgresql.conf
+COPY ./src/scripts/pg_hba.conf /etc/postgresql/14/main/pg_hba.conf
+
+# Default ENVs that can be overwritten
+ARG IASQL_ENV=local
+ENV IASQL_ENV=$IASQL_ENV
+ARG DB_USER=postgres
+ENV DB_USER=$DB_USER
+ARG DB_PASSWORD=test
+ENV DB_PASSWORD=$DB_PASSWORD
+
+EXPOSE 5432
```

Comment:
Eg:

```suggestion
EXPOSE 5432
EXPOSE 9229
```

Reply:
Committing this change but this is merely informative, the exposure is already being done in the `docker run` command with the port forwarding option `-p 9229:9229`.

Quoting from [`EXPOSE` docker docs](https://docs.docker.com/engine/reference/builder/#expose):
> The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the -p flag on docker run to publish and map one or more ports, or the -P flag to publish all exposed ports and map them to high-order ports.

Reply:
Huh. I wonder if it used to be required?